.evergreen/perf-send.sh

```diff
@@ -1,9 +1,11 @@
 #!/usr/bin/env bash
 
-set -euox pipefail
+set -euo pipefail
 
 source $DRIVERS_TOOLS/.evergreen/init-node-and-npm-env.sh
 
 TARGET_FILE=$(realpath "${TARGET_FILE:-./test/benchmarks/driver_bench/results.json}")
 
+set -o xtrace
```

Comment:
This is because the benchmark output used to be viewable when you clicked into just the task, but the xtrace for init-node has clobbered the benchmark results requiring you to click into parsley. 

For a quick glance it's nice if the results can be seen at the preview level, should get some of it back into view.